src/components/QuestionItem.js

```diff
@@ -9,6 +9,20 @@ function QuestionItem({ question }) {
     </option>
   ));
 
+  const handleDelete = (e) => {
+    const options = {
+      method: 'DELETE',
+      headers:
+      {
+        "Content-Type": "application/json",
+        Accept: "application/json"
+      }
+    }
+    fetch(`http://localhost:4000/questions/${id}`, options)
+    .then(_resp => e.target.parentElement.remove())
```

Comment:
Instead of, removing from the DOM with vanilla JS, you should be removing the object from state. If you were to create a new question, the "deleted" question would reappear I bet.